.github/workflows/integration_testing.yml

```diff
@@ -80,3 +81,82 @@ jobs:
           for folder in integration-tests/*/; do
             (cd "$folder" && fastn test --headless)
           done
+
+  slides-integration-test:
+    runs-on: ubuntu-latest
+
+    env:
+      FASTN_DB_URL: postgres://admin:admin@localhost:5432/fifthtry
+      FASTN_SECRET_KEY: fddfbbb85dcae750433875eb8f0fdced08d72aff
+
+    services:
+      postgres:
+        image: postgres:latest
+        env:
+          POSTGRES_DB: fifthtry
+          POSTGRES_USER: admin
+          POSTGRES_PASSWORD: admin
+        ports:
+          - 5432:5432
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
```

Comment:
I think we can reduce the interval to make this step faster. There is no harm is testing every second or even more frequently, it is our action, machine is not doing anything else etc.